proparse/src/main/java/org/prorefactor/proparse/antlr4/Lexer.java

```diff
@@ -219,56 +219,61 @@ ProToken nextToken() throws IOException {
           if (currChar == '=') {
             append();
             getChar();
-            return makeToken(ProParserTokenTypes.GTOREQUAL);
+            return makeToken(PreprocessorParser.GTOREQUAL);
           } else {
-            return makeToken(ProParserTokenTypes.RIGHTANGLE);
+            return makeToken(PreprocessorParser.RIGHTANGLE);
           }
 
         case '<':
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 12 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1151)